protocol/src/protocol.progress.proposed.ts

```diff
@@ -0,0 +1,62 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+'use strict';
+
+import { NotificationType, NotificationHandler } from 'vscode-jsonrpc';
+
+export interface WindowProgressClientCapabilities {
+	/**
+	 * Experimental client capabilities.
+	 */
+	experimental: {
```

Comment:
This should be a normal capability and nothing experimental. Since it is proposed we are sill free to change it. We should extens `_ClientCapabilities` with a additional optional property `windows` with an optional property `progress`.

Reply:
Okay, done :+1: